programs/ssl/ssl_base64_dump.c

```diff
@@ -17,8 +17,8 @@
 #define VER_MAJOR 0
 #define VER_MINOR 1
 
-static char debug = 0;
 FILE *b64_file = NULL;
+char debug = 0;
```

Comment:
For commit "Fix typos", could you instead not add the typos into the commits where the typos originated?